code/game/objects/items/grenades/flashbang.dm

```diff
@@ -34,8 +34,12 @@
 		M.Paralyze(20)
 		M.Knockdown(200)
 		M.soundbang_act(1, 200, 10, 15)
+		if HAS_TRAIT(M, TRAIT_FLASHBURN)
+			M.adjustFireLoss (150) //no coming back from that

```

Comment:
I mean, I think they technically could if they get left in darkness for a while (I think that that's fine, I'm just nitpicking the COMMENT here, not the thing being commented on).

Reply:
Not sure if this alpys but they have is a 1.5 modifier for burn but that may just be for weapons